synfig-core/src/modules/mod_magickpp/mptr_magickpp.cpp

```diff
@@ -128,33 +128,43 @@ magickpp_mptr::get_frame(synfig::Surface& surface, const synfig::RendDesc& /*ren
 		const auto height = image.size().height();
 		surface.set_wh(width, height);
 
-		image.type(Magick::TrueColorType);
-// Sadly Magick++ API is a mess with namespaces...
-{
-	using namespace Magick;
-	using namespace MagickCore;
+#if MagickLibVersion >= 0x701 // MAGICKCORE_CHECK_VERSION(7,0,0) does not work!
+		const bool has_alpha = image.alpha();
```

Comment:
I suggest moving the ImageType here as well. In this case, we group all the overrides together:
```
#if MagickLibVersion >= 0x701 // MAGICKCORE_CHECK_VERSION(7,0,0) does not work!
		const bool has_alpha = image.alpha();
		constexpr MagickCore::ImageType TrueColorAlphaType = Magick::TrueColorAlphaType;
#else
		const bool has_alpha = image.matte();
		constexpr MagickCore::ImageType TrueColorAlphaType = Magick::TrueColorMatteType;
#endif
```

Reply:
Done